### PR TITLE
changed key generation to deterministic

### LINKED ISSUE #7: Deterministic test generation improvements
After completing issue https://github.com/visoftsolutions/cairo-webauthn/issues/3. The test generation is mostly deterministic. There are two files that still change upon every script invocation:
Screenshot from 2023-09-06 10-23-13
We should modify the scripts generating these files so that their generation is deterministic.

### PR DESCRIPTION
**Pull request by verzotokumpel**

### CODE REVIEW

**Review by piniom — CHANGES_REQUESTED** [review]
The `src/auth/src/tests/verify_signature_gen_test.cairo` file still changes with every test generation. Please update the corresponding python file. 


Hint: 
The `get_random_string()` function is still used in `verify_signature_test.py`. Change it to the deterministic one.

**Review by piniom — CHANGES_REQUESTED** [review]
There are still changes in `verify_signature_gen_test.cairo` after running the script. Please fix it. 
Hint:
The changes are the result of calling `sk.sign(message, hashfunc=sha256)` instead of `sign_deterministic` in the `get_raw_signature` function in the `utils.py` file.

File: test_gen_scripts/utils.py [hunk 1]
```diff
@@ -41,11 +41,10 @@ def get_good_signature(message: bytes, hash=False):
     return (px, py, r, s)
 
 
-def get_raw_signature(message: bytes):
-    seed = generate_next_seed()
-    sk = generate_deterministic_key(seed)
+def get_raw_signature(message: bytes, sk):
```
Comment (ThePinion) [inline test_gen_scripts/utils.py:44]: Why is there `sk` as a function argument and what does it do?
It is shadowed here `sk = generate_deterministic_key(generate_next_seed())`

**Review by piniom — COMMENTED** [review]
Almost everything seems ok

**Pull request closed without merging**